src/app/layout/account-switcher.component.html

```diff
@@ -65,7 +65,7 @@
         ></i>
       </a>
     </div>
-    <div class="border"></div>
+    <div class="border" *ngIf="numberOfAccounts < 4 && numberOfAccounts > 0"></div>
```

Comment:
This can also just be `numberOfAccounts > 0` as the border should also be shown when the account limit message appears

Reply:
Fixed.